packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/servers/translations/Messages_fr_FR.json

```diff
@@ -1,4 +1,6 @@
 {
+  "dedicated_servers_all_servers": "All Servers",
```

Comment:
I think should be translate in French.

Reply:
Its pending from PO. I have already requested in the ticket.